t/to_words/to_words_ubi_8.4.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package               : to_words
+# Version               : v1.0.1
+# Source repo           : https://github.com/taimur-akhtar/to_words.git
+# Tested on             : UBI 8.4
+# Language              : Ruby
+# Travis-Check          : True
+# Script License        : Apache License, Version 2 or later
+# Maintainer            : Vikas . <kumar.vikas@in.ibm.com>
+#
+# Disclaimer            : This script has been tested in root mode on given
+# ==========              platform using the mentioned version of the package.
+#                         It may not work as expected with newer versions of the
+#                         package and/or distribution. In such case, please
+#                         contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e
```

Comment:
this will need to be update per the new template and move to the first line of the script